build/debian-iptables/cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+timeout: 10800s
+
+substitutions:
+  { "_ARCH": "amd64",
+    "_REGISTRY": "gcr.io/k8s-image-staging",
```

Comment:
is this the same as staging-k8s.gcr.io?

Reply:
No, this is the staging registry (it really is staging and not world-readable). staging-k8s.gcr.io is an alias for gcr.io/google-containers.